src/modules/LinearTracking.jl

```diff
@@ -180,7 +180,7 @@ function linear_bend_matrices(K0, L, gamma_0, e1=nothing, e2=nothing)
   return mx, my, r56, d, t
 end
 
-function linear_combined_matrices(K0, g, K1, L, gamma_0, e1=nothing, e2=nothing)
+function linear_combined_matrices(K0, L, gamma_0; g=K0, K1=0.0, e1=nothing, e2=nothing)
```

Comment:
```suggestion
function linear_dipole_matrices(K0, L, gamma_0; g=nothing, K1=nothing, e1=nothing, e2=nothing)
```

Instead of `K1` defaulting to `0.0` (which is a `Float64`), it should default to `nothing` so that users may take advantage of compile-time optimizations if the bend does not have any `K1`. Also note that we would want to specify quantities like this as `0` instead of `0.0` because `0` is an `Int64`, and `promote_type(Int64,Float32)=Float32)`. So if users want to user lower precision floats, they can. 

Also because a kicker is `g=0` with `K0 != 0`, I suggest renaming this to `dipole` so it is more general than just a bend. 

Finally I suggest not defaulting `g=K0` for synonymous behavior with `Beamlines`. Instead we let it be `nothing` so that for kicker elements (which have `g=0`) users can gain compile-time optimizations

Reply:
If you agree with my suggestions, then could you modify the source code to do similar `isnothing ... || .. == 0` for the `g` and the `K1`, like I do with the `e1` and `e2`?

Reply:
will do!